crates/ruff_linter/resources/test/fixtures/pyflakes/F401_33/__init__.py

```diff
@@ -0,0 +1,6 @@
+__all__ = ('Spam',)
```

Comment:
(you'll have to update the snapshots if you accept this change)

```suggestion
"""Regression test for https://github.com/astral-sh/ruff/issues/12897"""

__all__ = ('Spam',)
```

Reply:
Oops, good call. I'll accept these and rerun.